Makefile

```diff
@@ -10,7 +10,7 @@ ifeq (, $(shell which opentelemetry-collector-builder))
 	@{ \
 	set -e ;\
 	mkdir -p $(OTELCOL_BUILDER_DIR) ;\
-	curl -sLo $(OTELCOL_BUILDER) https://github.com/observatorium/opentelemetry-collector-builder/releases/download/v$(OTELCOL_BUILDER_VERSION)/opentelemetry-collector-builder_$(OTELCOL_BUILDER_VERSION)_linux_x86_64 ;\
+	curl -sLo $(OTELCOL_BUILDER) https://github.com/observatorium/opentelemetry-collector-builder/releases/download/v$(OTELCOL_BUILDER_VERSION)/opentelemetry-collector-builder_$(OTELCOL_BUILDER_VERSION)_linux_amd64 ;\
```

Comment:
What is the reason for the change here?

Reply:
This PR: https://github.com/observatorium/opentelemetry-collector-builder/pull/28

Reply:
Ok thanks.